CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.61.0] - 2022-07-19
+### Added
+- grpc - option to set custom maximum header list size.
```

Comment:
Please add change logs for tls support as well.

Reply:
This PR does not expose any new API for the user, changelog will be updated when we release it in #2141 #2145 etc.